CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Confluent's Python client for Apache Kafka
 
+## v1.8.0
+
+v1.8.0 is a maintenance release with the following fixes and enhancements:
+
+### Enhancements
+
+- Support rd_kafka_memberid() from python clients (#1154).
```

Comment:
Don't expose the librdkafka name here, but the python one, so something like:
` - Added consumer.memberid() method to retrieve the consumer's group member id (#1154)`